src/Shared/Controls/DebugLogs.xaml

```diff
@@ -0,0 +1,116 @@
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+	xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:ui="using:Microsoft.Toolkit.Uwp.UI"
+	xmlns:viewModels="using:StrixMusic.Sdk.ViewModels" xmlns:behaviors="using:OwlCore.WinUI.Behaviors"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+
+    <StackPanel
+		MaxWidth="1600"
+		Margin="20"
+		Spacing="4" VerticalAlignment="Top">
+
+
+        <TextBlock
+            MaxWidth="1000"
+			Margin="1,28,0,4"
+			Style="{StaticResource BodyStrongTextBlockStyle}"
+			Text="Debug Info" />
+
+        <labs:SettingsCard
+			Header="Window Size"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
+                    <Run Text="Height(px):"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.WindowHeight}"/>
+                    <Run Text=" Width(px):"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.WindowWidth}"/>
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:SymbolIcon Symbol=Clear}"
+			Header="Delete User Data"
+			HeaderIcon="{ui:SymbolIcon Symbol=Delete}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.DeleteUserDataCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			Header="Memory"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
+                    <Run Text="Memory Used:"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.MemoryUsage}"/>
+                    <Run Text="Expected Memory Limit:"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.TotalMemory}"/>
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:FontIcon Glyph=&#xE8A7;}"
+			Header="Open logs folder"
+			HeaderIcon="{ui:SymbolIcon Symbol=Folder}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.OpenLogFolderCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <TextBlock
+			Margin="1,28,0,4"
+			Style="{StaticResource BodyStrongTextBlockStyle}"
+			Text="Logs" />
+
+        <ScrollViewer x:Name="PART_SvLogs"
```

Comment:
Never put a ScrollViewer around a ListView. ListView has an internal scrollviewer already, and wrapping another one around it forces the ListView to an `auto` height.

At that point, everything is being shown and virtualization does not apply, but the ScrollViewer surrounding the ListView is hiding it.

Please remove this and use the internal ScrollViewer provided by `ListView`.